<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WeatherApp</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #1a2a6c, stop:1 #b21f1f);
    border-radius: 15px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: rgba(255, 255, 255, 0.1);
    border-radius: 15px;
    color: white;
    font-family: 'Segoe UI', Arial, sans-serif;
}

QLineEdit {
    background-color: rgba(255, 255, 255, 0.15);
    border: 2px solid rgba(255, 255, 255, 0.2);
    border-radius: 12px;
    padding: 10px 15px;
    color: white;
    font-size: 16px;
    selection-background-color: rgba(255, 255, 255, 0.3);
}

QLineEdit:focus {
    border: 2px solid rgba(255, 255, 255, 0.4);
    outline: none;
}

QPushButton {
    background-color: rgba(255, 255, 255, 0.2);
    border: 2px solid rgba(255, 255, 255, 0.3);
    border-radius: 12px;
    padding: 10px 25px;
    color: white;
    font-size: 16px;
    font-weight: 600;
    min-width: 100px;
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.3);
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 0.15);
}

QLabel {
    color: white;
    font-size: 16px;
}

QLabel#label_city {
    font-size: 28px;
    font-weight: bold;
    qproperty-alignment: AlignCenter;
}

QLabel#label_temp {
    font-size: 30px;
    font-weight: 300;
    qproperty-alignment: AlignCenter;
    margin: 20px 0;
}

QLabel#label_weather {
    font-size: 24px;
    qproperty-alignment: AlignCenter;
    margin-bottom: 30px;
}

QLabel#label_humidity, QLabel#label_wind {
    font-size: 18px;
    padding: 8px 0;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="inputLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">margin-right: 10px;</string>
        </property>
        <property name="placeholderText">
         <string>Enter city name...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_city">
      <property name="text">
       <string>—</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_temp">
      <property name="text">
       <string>—°</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_weather">
      <property name="text">
       <string>—</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0.1);
border-radius: 12px;
padding: 15px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_humidity">
         <property name="text">
          <string>Humidity: —%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_wind">
         <property name="text">
          <string>Wind Speed: — km/h</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
